src/stores/Account.ts

```diff
@@ -104,5 +107,11 @@ export const useAccountStore = createStore({
         setActiveCurrency(currency: CryptoCurrency) {
             this.state.activeCurrency = currency;
         },
+        setTour(tour: AccountState['tour']) {
+            this.state.tour = tour;
+        },
+        removeTour() {
+            this.state.tour = null;
+        },
```

Comment:
I don't think this method is necessary, plus its naming is not optimal. What prevents you from using `setTour(null)` wherever you currently use `removeTour()`?